docs/src/modules/components/AppSearch.js

```diff
@@ -63,8 +62,6 @@ function initDocsearch(userLanguage) {
       },
       // debug: true, // Set debug to true if you want to inspect the dropdown.
     });
-    // https://github.com/algolia/docsearch/issues/418
-    docsearchInput.setAttribute('aria-label', ariaLabel);
```

Comment:
Nice, I had a reminder in a few days from now to apply the change (7 days CDN cache). But it's not that important for delaying it much further :)